src/test/java/org/fundacionjala/trello/pages/card/CardPage.java

```diff
@@ -35,15 +49,41 @@ public boolean isDisplayed() {
         return true;
     }
 
-    public List<String> getAllCardNames(final String listName) {
-        String getCardsList = String.format(CARD_NAMES_LIST, listName);
-        action.waitForVisibilityOfAllElements(cardsList);
-        List<WebElement> getCards = driver.findElements(By.xpath(getCardsList));
+    public List<String> getCardNamesInList(final String listName) {
+        List<WebElement> getCards = getCardElementsInList(listName);
         List<String> list = new ArrayList<>();
         for (WebElement getCard : getCards) {
             String text = getCard.getText();
             list.add(text);
         }
         return list;
     }
+
+    public List<WebElement> getCardElementsInList(final String listName) {
+        String cardsInListLocator = String.format(GETS_ALL_CARDS_IN_A_LIST, listName);
+        action.waitForVisibilityOfAllElements(cardsList);
+        List<WebElement> getCards = driver.findElements(By.xpath(cardsInListLocator));
+        return getCards;
+    }
+
+    public FormPage<?> navigateToCard(final String listName, final String cardName) {
```

Comment:
make this return the specific class CardUpdateForm

Reply:
Done, Thank you